src/frontend/screens/Observation/index.tsx

```diff
@@ -61,7 +66,13 @@ export const ObservationScreen: NativeNavigationComponent<'Observation'> = ({
   const {lat, lon, createdBy} = observation;
   const isMine = deviceId === createdBy;
   // Currently only show photo attachments
-  const photos = [];
+  const attachmentUrls = useAttachmentUrlQueries(
+    observation.attachments,
+    'original',
+  );
+  const base64Uris = useAttachmentsBase64Query(
+    attachmentUrls.filter(r => !!r.data).map(r => r.data!),
+  );
```

Comment:
Why do you need to convert things to base64 data URIs? Why can't you pass the thumbnail URLs directly?

Reply:
This is weird one. I was struggling for good hour or so to make it work just by passing URL to a blob. However every time I did that the backend would return 400 so this can be considered as some kind of "workaround". If you have any idea why the backend may return 400 when requesting image via `<Image source={{uri: <ATTACHMENT URI>}}`, all suggestions are very welcome because I don't really like current solution cleanliness either.

Reply:
We discussed this offline and it seems like this is a necessity.